samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/FlipViewSamplePage.xaml

```diff
@@ -0,0 +1,95 @@
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:local="using:Uno.Toolkit.Samples.Content.Controls"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:msui="using:Microsoft.UI.Xaml.Controls"
+	xmlns:sample="using:Uno.Toolkit.Samples"
+	xmlns:utu="using:Uno.Toolkit.UI"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	mc:Ignorable="d">
+
+	<sample:SamplePageLayout IsDesignAgnostic="True">
+		<sample:SamplePageLayout.DesignAgnosticTemplate>
+			<DataTemplate>
+				<StackPanel Margin="0,20" Spacing="20">
+
+					<StackPanel.Resources>
+						<ResourceDictionary>
+							<Style TargetType="x:TextBlock">
+								<Setter Property="HorizontalAlignment" Value="Center" />
+							</Style>
+						</ResourceDictionary>
+					</StackPanel.Resources>
+
+					<TextBlock Text="Try to navigate throw the FlipView using the buttons inside it. " />
+
+					<FlipView
+						x:Name="flipView"
+						utu:SelectorExtensions.PipsPager="{Binding ElementName=pipsPager}"
+						Background="Blue">
+						<FlipView.Items>
+							<Grid Background="Azure">
+								<Button
+									HorizontalAlignment="Left"
+									utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
+									Content="Previous" />
+								<Button
+									HorizontalAlignment="Right"
+									utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
+									Content="Next" />
+							</Grid>
+							<Grid Background="Blue">
+								<Button
+									HorizontalAlignment="Left"
+									utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
+									Content="Previous" />
+								<Button
+									HorizontalAlignment="Right"
+									utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
+									Content="Next" />
+							</Grid>
+							<Grid Background="Yellow">
+								<Button
+									HorizontalAlignment="Left"
+									utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
+									Content="Previous" />
+								<Button
+									HorizontalAlignment="Right"
+									utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
+									Content="Next" />
+							</Grid>
+							<Grid Background="Fuchsia">
+								<Button
+									HorizontalAlignment="Left"
+									utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
+									Content="Previous" />
+								<Button
+									HorizontalAlignment="Right"
+									utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
+									Content="Next" />
+							</Grid>
+						</FlipView.Items>
+					</FlipView>
+
+					<msui:PipsPager
+						x:Name="pipsPager"
+						MinHeight="100"
+						HorizontalAlignment="Center"
+						MaxVisiblePips="10" />
+
+					<TextBlock Text="To add a new view, press the button bellow" />
+
+					<Button
+						x:Name="btn"
```

Comment:
```suggestion
						x:Name="AddNewPageButton"
```
the x:Name can be a bit more descriptive, so when you are referring it in code-behind or unit-tests you dont have to check the xaml to check what it is.